src/util/execute-lifecycle-script.js

```diff
@@ -227,6 +219,11 @@ export async function makeEnv(
 
       pathParts.unshift(`${dependencyInformation.packageLocation}/.bin`);
     }
+
+    // Note that NODE_OPTIONS doesn't support any style of quoting its arguments at the moment
```

Comment:
Referring to your PR on Node would be nice. Also, would be great if we can centralize some of this PnP path logic.

Reply:
I think I'll keep it duplicated for now - since it's very few lines, I'm concerned about over-abstracting it (since it's only used in two places and there are no reasons to think it will change)